app/renderer/components/preferences/payment/bitcoinDashboard.js

```diff
@@ -341,7 +341,7 @@ class BitcoinDashboard extends ImmutableComponent {
                         {/* TODO: refactor button.js */}
                         <Button className={cx({
                           primaryButton: true,
-                          [css(globalStyles.spacing.dialogInsideMargin)]: true
+                          [css(styles.bitcoinAddressButton)]: true
```

Comment:
we can change a name, but one problem with this name is that is really generic and we only need margin bottom here. What happens if you would need margin top and bottom in this component?